makefiles/simulators/Makefile.nvc

```diff
@@ -8,6 +8,7 @@ clean::
 
 else
 
+#TRACE :=--vhpi-trace
```

Comment:
@Martoni would it be better to enable the TRACE based on an env var? COCOTB_NVC_TRACE for instance rather than needing the modification of a Makefile?

Also thanks for the all the work that you have clearly been putting in of late.